Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.ci.openshift.org/stolostron/builder:go1.17-linux AS builder
+
+ARG REMOTE_SOURCE
+ARG REMOTE_SOURCE_DIR
+
+COPY $REMOTE_SOURCE $REMOTE_SOURCE_DIR/app/
+WORKDIR $REMOTE_SOURCE_DIR/app
+
+# compile go tests in build image
+RUN GOFLAGS="" go get -u github.com/onsi/ginkgo/v2/ginkgo@v2.1.0
+RUN GOFLAGS="" ginkgo build pkg/tests/placeholder
+
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+RUN microdnf update && \
+    microdnf clean all
+
+# expose env vars for runtime
+ENV KUBECONFIG "/opt/.kube/config"
+ENV IMPORT_KUBECONFIG "/opt/.kube/import-kubeconfig"
+ENV OPTIONS "/resources/options.yaml"
+
+# install ginkgo into built image
+COPY --from=builder /go/bin/ /usr/local/bin
+
+# install operator binary
+COPY --from=builder $REMOTE_SOURCE_DIR/app/pkg/tests/placeholder/placeholder.test /test/placeholder/placeholder.test
+
+VOLUME /results
+WORKDIR "/test"
+
+CMD ["/bin/bash", "-c", "ginkgo placeholder/placeholder.test -- --ginkgo.trace --ginkgo.v --ginkgo.junit-report=/results/result-managed-serviceaccount-placeholder.xml"]
```

Comment:
ginkgo v2 report has a `[IT]` label in each test case, will need to work with CICD to see if that's ok. Otherwise, we may need to filter out that label with an `sed` command 😂

Reply:
example report results:
```
<?xml version="1.0" encoding="UTF-8"?>
  <testsuites tests="1" disabled="0" errors="0" failures="0" time="0.0010518">
      <testsuite name="Manged-Serviceacount Placeholder Suite" package="/test/placeholder" tests="1" disabled="0" skipped="0" errors="0" failures="0" time="0.0010518" timestamp="2022-01-26T19:38:44">
          <properties>
              <property name="SuiteSucceeded" value="true"></property>
              <property name="SuiteHasProgrammaticFocus" value="false"></property>
              <property name="SpecialSuiteFailureReason" value=""></property>
              <property name="SuiteLabels" value="[]"></property>
              <property name="RandomSeed" value="1643225924"></property>
              <property name="RandomizeAllSpecs" value="false"></property>
              <property name="LabelFilter" value=""></property>
              <property name="FocusStrings" value=""></property>
              <property name="SkipStrings" value=""></property>
              <property name="FocusFiles" value=""></property>
              <property name="SkipFiles" value=""></property>
              <property name="FailOnPending" value="false"></property>
              <property name="FailFast" value="false"></property>
              <property name="FlakeAttempts" value="0"></property>
              <property name="EmitSpecProgress" value="false"></property>
              <property name="DryRun" value="false"></property>
              <property name="ParallelTotal" value="1"></property>
              <property name="OutputInterceptorMode" value=""></property>
          </properties>
          <testcase name="[It] Server-foundation: [P1][Sev1][server-foundation] managed-serviceaccount placeholder" classname="Manged-Serviceacount Placeholder Suite" status="passed" time="0.0007131"></testcase>
      </testsuite>
  </testsuites>
```